cli/config/configuration.toml.in

```diff
@@ -140,6 +140,12 @@ enable_iothreads = @DEFENABLEIOTHREADS@
 # Default false
 #use_vsock = true
 
+# VFIO devices are hotplugged on a bridge by default. 
+# Enable hotplugging on root bus. This may be required for devices with
+# a large PCI bar, as this is a current limitation with hotplugging on 
+# a bridge.
```

Comment:
What is the default value?
This won't be possible on machine types based on PCIe, you should add a comment explaining why and what machines types are not compatible

Reply:
@devimc I have added a comment.